Makefile

```diff
@@ -30,22 +30,26 @@ build:
 	docker build --target build --network nuodb-net -f dockers/centos/Dockerfile -t nuodb/node-nuodb:$(VERSION)-build .
 
 #:help: test        | Runs the `test` target, building and testing the driver.
+#changed to properly create and mount volumes
 .PHONY: test
 test: build
 	docker volume create cores
-	docker run -it --cap-add=SYS_PTRACE --memory 1g --volume $(CURDIR)/cores:/cores --volume $(CURDIR)/valgrind:/valgrind --name test --rm --network nuodb-net nuodb/node-nuodb:$(VERSION)-build npm run test
+	docker volume create valgrind
```

Comment:
Should the `clean` target remove these volumes?

Reply:
It probably should, as we discussed I think the entirety of `make clean` needs a rework so it doesn't completely nuke you docker.